libsplinter/src/network/auth/handlers.rs

```diff
@@ -120,10 +128,31 @@ impl Handler for AuthorizedHandler {
         context: &MessageContext<Self::Source, Self::MessageType>,
         _sender: &dyn MessageSender<Self::Source>,
     ) -> Result<(), DispatchError> {
-        debug!("Connection {} authorized", context.source_connection_id());
+        debug!(
+            "Received authorize message from {}",
+            context.source_connection_id()
+        );
+        match self.auth_manager.next_state(
+            context.source_connection_id(),
+            AuthorizationAction::RemoteAuthorizing,
+        ) {
+            Err(err) => {
+                debug!(
```

Comment:
should this be warn?

Reply:
should this cause the connection to close?

Reply:
The connection manager will get the `Unauthorized` result, and close the connection.

Reply:
Possibly those should all be `warn`. I'll add a commit for that.